Formula/libchewing.rb

```diff
@@ -0,0 +1,47 @@
+class Libchewing < Formula
+  desc "Intelligent phonetic input method library"
+  homepage "http://chewing.im/"
+  url "https://github.com/chewing/libchewing/releases/download/v0.5.1/libchewing-0.5.1.tar.bz2"
+  sha256 "9708c63415fa6034435c0f38100e7d30d0e1bac927f67bec6dfeb3fef016172b"
+  head "https://github.com/chewing/libchewing.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "texinfo" => :build
+  depends_on "sqlite"
+
+  def install
+    if build.head?
```

Comment:
You've wrapped this in a `build.head?` but you've left all the autotool deps for both stable & head. Intentional or?